Test_App/src/main/java/Test_App/Transaction.java

```diff
@@ -24,6 +29,9 @@ public Transaction getCurrentTransaction() {
     public String getDate() {
         return date;
     }
+    public Date getRecordDate() throws ParseException {
+        return new SimpleDateFormat("yyyy/MM/dd HH:mm").parse(this.getDate());
```

Comment:
The format string should be extracted to a constant as it is used in more than one place and we expect it to be the same in both places.